Makefile

```diff
@@ -192,7 +192,7 @@ lint: golangci-lint
 
 # Generate code
 .PHONY: generate
-generate: controller-gen
+generate: api-docs controller-gen
```

Comment:
the `api-docs` traget was remvoed from generate to improve developer UX by minimizing local changes when developing the operator.  #2421

Reply:
Was that actually necessary? `api-docs` is a noop if you haven't made any changes impacting the docs, so it should be fine to have it here.

Reply:
What does the generate update? Is it the deepcopy? 

I think the api-docs generate should be bundled with the target to generate CRD

Reply:
`generate` only generates the deepcopy code, yes.

Reply:
But I do agree that `api-docs` should be a part of `bundle` rather than `generate`.